src/components/UserSummary.js

```diff
@@ -0,0 +1,253 @@
+import React, { Fragment } from 'react'
+import { Card, Image, Grid, Label, Tab, Embed, Icon } from 'semantic-ui-react'
+import { RingLoader } from 'react-spinners'
+import moment from 'moment'
+import '../assets/UserSummary.css'
+
+const UserSummary = props => {
+  let { user } = props
+  if (user) {
+    const usersProjectsList = user.contributions
```

Comment:
This maybe should be called `latestUserContributionList` or something of the sorts, because it is not the `usersProjectList`